test/DOMINO/test_domino.py

```diff
@@ -25,57 +25,75 @@ class TestDOMINO:
 
     def test_domino_required(self):
         # Only include required arguments
-        out_path = Path(OUT_FILE_DEFAULT)
-        out_path.unlink(missing_ok=True)
+        OUT_FILE_DEFAULT.unlink(missing_ok=True)
         DOMINO.run(
-            network=TEST_DIR+'input/domino-network.txt',
-            active_genes=TEST_DIR+'input/domino-active-genes.txt',
+            network=TEST_DIR / 'input' / 'simple' / 'domino-network.txt',
+            active_genes=TEST_DIR / 'input' / 'simple' / 'domino-active-genes.txt',
             output_file=OUT_FILE_DEFAULT)
         # output_file should be empty
-        assert out_path.exists()
+        assert OUT_FILE_DEFAULT.exists()
 
     def test_domino_optional(self):
         # Include optional arguments
-        out_path = Path(OUT_FILE_OPTIONAL)
-        out_path.unlink(missing_ok=True)
+        OUT_FILE_DEFAULT.unlink(missing_ok=True)
         DOMINO.run(
-            network=TEST_DIR+'input/domino-network.txt',
-            active_genes=TEST_DIR+'input/domino-active-genes.txt',
+            network=TEST_DIR / 'input' / 'simple' / 'domino-network.txt',
+            active_genes=TEST_DIR / 'input' / 'simple' / 'domino-active-genes.txt',
             output_file=OUT_FILE_OPTIONAL,
             slice_threshold=0.4,
             module_threshold=0.06)
         # output_file should be empty
-        assert out_path.exists()
+        assert OUT_FILE_OPTIONAL.exists()
 
     def test_domino_missing_active_genes(self):
         # Test the expected error is raised when active_genes argument is missing
         with pytest.raises(ValueError):
             # No active_genes
             DOMINO.run(
-                network=TEST_DIR+'input/domino-network.txt',
+                network=TEST_DIR / 'input' / 'simple' / 'domino-network.txt',
                 output_file=OUT_FILE_DEFAULT)
 
     def test_domino_missing_network(self):
         # Test the expected error is raised when network argument is missing
         with pytest.raises(ValueError):
             # No network
             DOMINO.run(
-                active_genes=TEST_DIR+'input/domino-active-genes.txt',
+                active_genes=TEST_DIR / 'input' / 'simple' / 'domino-active-genes.txt',
                 output_file=OUT_FILE_DEFAULT)
 
+    def test_domino_small(self):
```

Comment:
Does the existing simple test already cover the same condition? It has a comment "output_file should be empty". We may instead need the "required" test to use a larger input so this one can be the small input, empty output test. That could wait for a follow up PR because it is out of scope.

Reply:
I think we have a larger network that seems to work for DIAMOnD that we may be able to reuse here - though I agree that that should be a separate PR.